docs/source/schema-design/federated-schemas/entities/intro.mdx

```diff
@@ -54,6 +54,12 @@ To define an entity within a particular subgraph, you do the following:
 1. Apply the [`@key` directive](#1-define-a-key) to an object type.
 2. Define the object type's [reference resolver](#2-define-a-reference-resolver).
 
+<Tip>
+
+With Apollo Connectors, you add [connector directives](/graphos/schema-design/connectors/directives) instead of writing reference resolver code. 
```

Comment:
Should we also mention the necessity to mark entities as `entity:true` in a connectors schema?

Reply:
yes we can that add that as a detail:

```suggestion
With Apollo Connectors, you add [connector directives](/graphos/schema-design/connectors/directives) instead of writing reference resolver code. 

You can [set `entity: true`](/graphos/schema-design/connectors/directives#rules-for-entity-true) for the `@connect` directive to provide an entity resolver for its fields.
```